src/bust.py

```diff
@@ -311,6 +313,7 @@ def get_google_form_url(self, image_url: Optional[str] = None) -> Optional[str]:
         return form_url
 
     async def send_stats(self, interaction: Interaction) -> None:
+        interaction.defer(ephemeral=True)
```

Comment:
This is just for my own understanding, but why are we not awaiting this defer like for `play` and `create_controller`?